docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -2,24 +2,29 @@
 Secure internal communication
 =============================
 
-The Trino cluster can be configured to use secured communication with internal
-authentication of the nodes in the cluster, and optionally added security with
-:ref:`TLS <glossTLS>`.
+You must configure your Trino cluster to use a shared secret to secure
```

Comment:
That implies that is is required.. which is not true.. only required if you want to use any authentication or if you want internal TLS

Reply:
"Absolutely required" is what I kept asking you, and you kept saying yes ...

Reply:
well.. it is not required if you have no authentication and do not want to use internal TLS... thats what I tried to communicate .. essentially its required in nearly all real world usage scenarios..

Reply:
✅